packages/jsonresume-theme-standard/src/index.js

```diff
@@ -1,16 +1,12 @@
 import { renderToString } from 'react-dom/server';
 import { ServerStyleSheet } from 'styled-components';
-import Document from './ui/Document';
 import Resume from './ui/Resume';
 
 export const render = (resume) => {
   const sheet = new ServerStyleSheet();
   const html = renderToString(sheet.collectStyles(<Resume resume={resume} />));
-  const body = <Resume resume={resume} />;
-  const styles = sheet.getStyleElement();
-  sheet.seal();
-  console.log({ styles });
-  return renderToString(<Document body={body} styles={styles} />);
+  const styles = sheet.getStyleTags();
+  return `<!DOCTYPE html><head>${styles}</head><body>${html}</body></html>`;
```

Comment:
Simplified this. The old version violates the eslint rules and then I tried to fix things, it broke. I also could not really follow the logic. Now I do understand better what is going on.